### PR TITLE
[SWE-1184] [BRANDBOOK] Create Header #done

### PR DESCRIPTION
**Pull request by AnastasiaShapowalova**
Screenshot 2022-07-19 at 12 26 30

Screenshot 2022-07-19 at 12 39 55

### CODE REVIEW

File: src/components/elements/Icon/Icon.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+const Icon = (props) => {
+  const { src, alt, className } = props
+
+  return <img src={src} alt={alt} className={className} />
+}
+Icon.propTypes = {
+  src: PropTypes.string,
```
Comment (5P4C3GR33D) [inline src/components/elements/Icon/Icon.js:10]: Not all propTypes are written here.

File: src/components/elements/Typography/Title/Title.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      [typographyStyles['text-secondary']]: secondary,
+      [className]: className
+    }),
+    dangerouslySetInnerHTML: { __html: children },
+    ...rest
+  })
+}
+
+Title.propTypes = {
+  children: PropTypes.string,
```
Comment (5P4C3GR33D) [inline src/components/elements/Typography/Title/Title.js:35]: Same here.

File: src/components/layouts/Header/Header.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+          </div>
+        )}
+      </div>
+    </div>
+  )
+}
+
+Header.propTypes = {
+  breadcrumbs: PropTypes.array,
+  breadcrumbsProps: PropTypes.object
```
Comment (olenashevchuk) [inline src/components/layouts/Header/Header.js:32]: delete unused prop

Comment (5P4C3GR33D) [inline src/components/layouts/Header/Header.js:32]: One propType is useless.

File: src/components/layouts/Header/Header.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+import Logo from 'assets/senseteqLogo.svg'
+import PropTypes from 'prop-types'
+import React from 'react'
+
+const Header = (props) => {
+  const { breadcrumbs } = props
+
+  const breadcrumbsComputed =
+    Array.isArray(breadcrumbs) && breadcrumbs.join(' — ')
```
Comment (olenashevchuk) [inline src/components/layouts/Header/Header.js:11]: use just ? for condition

File: src/components/layouts/Header/Header.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+  return (
+    <div className="container">
+      <div className="row align-items-center justify-content-between">
+        {breadcrumbsComputed ? (
```
Comment (5P4C3GR33D) [inline src/components/layouts/Header/Header.js:16]: Better this way:
```
<div className="col-auto">
  {breadcrumbsComputed ? (
    <Text {...breadcrumbsComputed}>{breadcrumbsComputed}</Text>
   ) : (
     <Icon src={Logo} alt="Senseteq Logo" />
   )}
 </div>
```

**Review by olenashevchuk — APPROVED** [review]

**Review by olenashevchuk — APPROVED** [review]

**Review by 5P4C3GR33D — APPROVED** [review]

**Pull request merged**